get_stories.php

```diff
@@ -18,13 +18,14 @@ public static function nprstory_cron_pull() {
 		//because this is run on cron, and may be fired off by an non-admin, we need to load a bunch of stuff
 		require_once (ABSPATH . '/wp-admin/includes/file.php');
 		require_once (ABSPATH . '/wp-admin/includes/media.php');
-		require_once (ABSPATH . '/wp-admin/includes/admin.php');
-		require_once (ABSPATH . '/wp-includes/class-wp-error.php');
```

Comment:
`/wp-cron.php` [loads](https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-cron.php#L26) `/wp-load.php`
`/wp-load.php` either [loads `/wp-settings.php` directly](https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-load.php#L39) or loads `wp-config.php`which loads `/wp-settings.php` - in either case  `/wp-settings.php` is loaded
 `/wp-settings.php` [loads](https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-settings.php#L83) `WPINC . '/class-wp-error.php'`

Thus we don't need to load `/wp-includes/class-wp-error.php`

`/wp-admin/includes/admin.php` was not required in testing of this section.

`/wp-admin/includes/file.php` is required to access the API.

`/wp-admin/includes/media.php` is required for image handling.